sdk/kfp-tekton-limitation.md

```diff
@@ -0,0 +1,9 @@
+
+## kfp-tekton limitation 
```

Comment:
Can we add a header and group the issues/limitations based on the features? such as `###Affinity, Node Selector, and Tolerations` and `###Sidecars`

Reply:
ok